<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCheckBox" name="checkBox_v1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>v1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s12">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>s12</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>320</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>s11</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_tanque1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>61</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_tanque2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s22">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>100</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>s22</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s21">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>170</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>s21</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_bomba">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>70</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>bomba</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_tanque3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>140</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_v2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>v2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s31">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>s31</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_s32">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>130</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>s32</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>664</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
